lbc.tex

```diff
@@ -1,16 +1,16 @@
 \chapter{Lateral Boundary Conditions}
 \label{lbc_chap}
 
-Several lateral boundary condition options exist for the ARW 
-that are suitable for idealized flows, and a specified lateral boundary
-condition for real-data simulations is available.  These choices are
+The ARW provides several lateral boundary conditions suitable 
+for idealized flow, in addition to a specified lateral boundary
+condition for real-data simulations.  These choices are
 handled via a run-time option in the Fortran namelist file.
 For nesting, all fine domains use the 
 nest time-dependent lateral boundary condition where the outer row and column of the
```

Comment:
@davegill For a nest, do we specify a single row and column or two of them by the parent domain data?

Reply:
@weiwangncar
We use one row / column for the `spec_zone` for both the `nested` and `specified` LBC options.
```
     IF (specified) THEN

       ! Arrays for specified boundary conditions

       DO loop = spec_zone + 1, spec_zone + relax_zone
         fcx(loop) = 0.1 / dt * (spec_zone + relax_zone - loop) / (relax_zone - 1)
         gcx(loop) = 1.0 / dt / 50. * (spec_zone + relax_zone - loop) / (relax_zone - 1)
         spongeweight=exp(-(loop-(spec_zone + 1))*spec_exp)
         fcx(loop) = fcx(loop)*spongeweight
         gcx(loop) = gcx(loop)*spongeweight
       ENDDO

     ELSE IF (nested) THEN

       ! Arrays for specified boundary conditions

       DO loop = spec_zone + 1, spec_zone + relax_zone
         fcx(loop) = 0.1 / dt * (spec_zone + relax_zone - loop) / (relax_zone - 1)
         gcx(loop) = 1.0 / dt / 50. * (spec_zone + relax_zone - loop) / (relax_zone - 1)
       ENDDO

     ENDIF
```